composer.json

```diff
@@ -6,20 +6,17 @@
     "license": "BSD-3-Clause",
     "require": {
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-core": "4.5.x-dev",
-        "cwp/cwp-core": "2.5.x-dev",
-        "silverstripe/auditor": "2.2.x-dev",
-        "silverstripe/environmentcheck": "2.1.x-dev",
-        "silverstripe/hybridsessions": "2.2.x-dev",
-        "silverstripe/mimevalidator": "2.0.x-dev"
+        "silverstripe/recipe-core": "^4",
```

Comment:
CWP is constrained to specific minor releases - this needs to be more specific than this

Reply:
yes, however we don't need to make that decision for the next major just yet?

Reply:
Have you already made a major version branch aside from this one? CWP is not going to have a new major version released, which is one of the reasons we never created branches implying it might

Reply:
yeah, a major version branch should follow when this is merged. Since I cannot make PRs for creating branches, I wanted this to be reviewed first.